components/bl-clock/src/stopwatch.js

```diff
@@ -1,20 +1,51 @@
 import { useEffect, useRef, useState } from 'react';
 
+const TimeVariant = {
+  HHMMSS: 'hhmmss',
+  MMSS: 'mmss',
+  SS: 'ss',
+};
+
+const stopwatchMap = {
```

Comment:
Try to find meaningful variable name

There should be an entity with which we work:
time/string/timestamp/some kind of object

and action that we do on thin data:
resolve - Resolver, format - Formatter, and so on. Composer, Converter, Transformer...